crates/networking/p2p/src/network/lean.rs

```diff
@@ -128,36 +137,189 @@ impl LeanNetworkService {
                 .build()
         };
 
-        Ok(LeanNetworkService { lean_chain, swarm })
+        Ok(LeanNetworkService {
+            lean_chain,
+            network_config,
+            swarm,
+            peer_table: parking_lot::RwLock::new(HashMap::new()),
```

Comment:
```suggestion
            peer_table: RwLock::new(HashMap::new()),
```

can't we remove the qualifer

Reply:
there are two RwLock's in this file used in different places...so importing both causes a conflict